.github/workflows/build.yml

```diff
@@ -16,6 +16,14 @@ jobs:
     name: Build
     runs-on: ubuntu-latest
     steps:
-    - uses: actions/checkout@v1
-    - name: Build the Docker image
-      run: docker build .
+      - name: checkout code
+        uses: actions/checkout@v2
+      - name: install buildx
+        id: buildx
+        uses: crazy-max/ghaction-docker-buildx@v1
```

Comment:
Could you explain why do we need `buildx` ?

Reply:
buildx is an experimental docker client tool for multiarch builds on a single arch. 

@shahidhs-ibm can you talk to your colleagues about ppc64le support? It would be great to get a single PR for both Z and Power...

Reply:
FYI, We've use buildx on z for other open source project and seems to be working quite well. Regarding power, it's done by different team, probably won't PR both together, testing on platform could face different problem too.

Reply:
fair point -- I'm learning that vm and hardware access is a different challenge / schedule for each platform, with Z arriving before P, it seems.